prime-router/src/main/kotlin/fhirengine/translation/TranslationSchemaManager.kt

```diff
@@ -22,8 +23,7 @@ class TranslationSchemaManager : Logging {
 
     companion object {
 
-        private val context = DefaultHapiContext()
-        private val hl7Parser = context.getGenericParser()
+        private val hL7Reader = HL7Reader(ActionLogger())
```

Comment:
This change was required due a series of confusions:

- the tests from the initial work that I had forked the last PR did not have assertions that covered whether or not the schemas were valid; they merely checked that validation had occurred
  - This is also why the input files have all changed as they actually weren't valid to begin with
- After the code was actually checking that input matched the output I encountered an issue on the HL7 test where the input/output were identical with the exception that the converted was using `\r` for new lines and the expected output was using `\n` which means they were not equal.  It probably is worth investigating at some point what is causing the difference, but for now I opted to use the same reader class for parsing HL7 strings that the production code uses